akd/src/append_only_zks.rs

```diff
@@ -315,14 +315,14 @@ impl Azks {
         let longest_prefix = lcp_node.label;
         // load with placeholder nodes, to be replaced in the loop below
         let mut longest_prefix_children = [Node {
-            label: NodeLabel::root(),
-            hash: crate::utils::empty_node_hash::<H>(),
+            label: EMPTY_LABEL,
+            hash: crate::utils::empty_node_hash_no_label::<H>(),
         }; ARITY];
         let state = lcp_node.get_state_at_epoch(storage, epoch).await?;
-
         for (i, child) in state.child_states.iter().enumerate() {
             match child {
                 None => {
+                    println!("i = {}, empty", i);
```

Comment:
Please stick to `debug!` or `warn!` etc traces. They do proper formatting, and can redirect to other outputs (e.g. file writing) if needed. If this was just for debugging it should get removed before being landed. Same goes for the other println! statements below